cypress/e2e/requests/filter-request-by-tags.cy.js

```diff
@@ -0,0 +1,74 @@
+import testType from '../../support/dictionary/testTypes';
+import TopMenu from '../../support/fragments/topMenu';
+import Requests from '../../support/fragments/requests/requests';
+import Users from '../../support/fragments/users/users';
+import DevTeams from '../../support/dictionary/devTeams';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import { ITEM_STATUS_NAMES, REQUEST_TYPES } from '../../support/constants';
+
+describe('ui-requests: Filter requests by tags', () => {
+  const userIds = [];
+  const requests = [];
+  const instances = [];
+  const requestTypes = { PAGE: 'Page', HOLD: 'Hold', RECALL: 'Recall' };
+  let oldRulesText;
+  let requestPolicyId;
+  const tags = ['[', '(', 'test_aqa'];
+
+  beforeEach(() => {
+    cy.loginAsAdmin();
+    cy.getAdminToken().then(() => {
+      Requests.setRequestPolicyApi().then(({ oldRulesAsText, policy }) => {
+        oldRulesText = oldRulesAsText;
+        requestPolicyId = policy.id;
+      });
+      Object.values(requestTypes).forEach((requestType) => {
+        const itemStatus =
+          requestType === REQUEST_TYPES.PAGE
+            ? ITEM_STATUS_NAMES.AVAILABLE
+            : ITEM_STATUS_NAMES.CHECKED_OUT;
+        Requests.createRequestApi(itemStatus, requestType).then(
+          ({ instanceRecordData, createdRequest, createdUser }) => {
+            userIds.push(createdUser.id);
+            instances.push(instanceRecordData);
+            requests.push(createdRequest);
+          },
+        );
+      });
+    });
+  });
+
+  afterEach(() => {
+    instances.forEach((instance) => {
+      cy.deleteItemViaApi(instance.itemId);
+      cy.deleteHoldingRecordViaApi(instance.holdingId);
+      InventoryInstance.deleteInstanceViaApi(instance.instanceId);
+    });
+    requests.forEach((request) => {
+      Requests.deleteRequestViaApi(request.id);
+    });
+    userIds.forEach((id) => {
+      Users.deleteViaApi(id);
+    });
+  });
+
+  it('C9320 Filter requests by tags', { tags: [testType.smoke, DevTeams.vega] }, () => {
```

Comment:
Test description should ends with devTeam: " (vega)"

Reply:
done, thank you